backend/daemon/storage/schema.sql

```diff
@@ -81,6 +81,20 @@ FROM structural_blobs
 JOIN blobs ON blobs.id = structural_blobs.id
 JOIN resources ON structural_blobs.resource = resources.id;
 
+-- View blobs metadata
+CREATE VIEW meta_view AS
+SELECT sb.meta, resources.iri, public_keys.principal
+FROM structural_blobs sb
+JOIN public_keys ON public_keys.id = sb.author
+JOIN resources ON resources.id = sb.resource
+JOIN (
```

Comment:
Why joining a subquery?

I think you'd get the same result with:

```sql
SELECT sb.meta, resources.iri, public_keys.principal, MAX(sb.ts)
FROM structural_blobs sb
JOIN public_keys ON public_keys.id = sb.author
JOIN resources ON resources.id = sb.resource
WHERE type= 'Change' AND meta IS NOT NULL
GROUP BY resource
```

or

```sql
SELECT sb.meta, resources.iri, public_keys.principal
FROM structural_blobs sb
JOIN public_keys ON public_keys.id = sb.author
JOIN resources ON resources.id = sb.resource
WHERE type= 'Change' AND meta IS NOT NULL
GROUP BY resource HAVING sb.ts = MAX(sb.ts)
```

Reply:
Actually, I realized it's more involved since that didn't catch documents (or drafts) with no title. Now they pass all the tests